### PR TITLE
Initialise the application with spring-boot-template

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-572

### Change description ###
Initialise the application from spring-boot-template.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+server:
+  port: 8484
+
+management:
+  endpoints:
+    web:
+      base-path: /
+
+#If you use a database then uncomment below lines and update db properties accordingly
```
Comment (rkondratowicz) [inline src/main/resources/application.yaml]: looks like this could be removed

File: infrastructure/aat.tfvars [hunk 2]
```diff
@@ -0,0 +1 @@
+capacity = "2"
```
Comment (nitinprabhu) [inline infrastructure/aat.tfvars]: capacity can be default 1

Comment (nitinprabhu) [reply infrastructure/aat.tfvars]: so we can remove this

File: infrastructure/main.tf [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  product             = "${var.product}-${var.component}"
+  location            = "${var.location_app}"
+  env                 = "${var.env}"
+  ilbIp               = "${var.ilbIp}"
+  subscription        = "${var.subscription}"
+  capacity            = "${var.capacity}"
+  common_tags         = "${var.common_tags}"
+
+  app_settings = {
+    LOGBACK_REQUIRE_ALERT_LEVEL = "false"
```
Comment (nitinprabhu) [inline infrastructure/main.tf]: is this lines required anymore ?

Comment (aliveni) [reply infrastructure/main.tf]: This is exactly the code generated by the spring boot template. We can update it when needed

Comment (nitinprabhu) [reply infrastructure/main.tf]: this is not required any more

Comment (aliveni) [reply infrastructure/main.tf]: ok, I'll remove

Comment (doncem) [reply infrastructure/main.tf]: yep, non of those 2 are needed

File: src/main/resources/application.yaml [hunk 4]
```diff
@@ -6,7 +6,6 @@ management:
     web:
       base-path: /
```
Comment (doncem) [inline src/main/resources/application.yaml]: iirc, pipeline will start passing with these changes present:
```
management:
  endpoint:
    health:
      show-details: "always"
  endpoints:
    web:
      base-path: /
```

Comment (aliveni) [reply src/main/resources/application.yaml]: I'll create another PR for pipeline changes.

File: infrastructure/aat.tfvars (hunk 2 above)
Comment (aliveni) [reply infrastructure/aat.tfvars]: Removed

**Pull request merged**